Showcase-iOS/ShowcaseError/View/ErrorView.swift

```diff
@@ -1,63 +1,53 @@
-//
-//  ErrorView.swift
-//  Showcase-iOS
-//
-//  Created by Kagiso Mohajane on 2018/05/16.
-//  Copyright © 2018 DVT. All rights reserved.
-//
-
 import UIKit
 
 class ErrorView: UIView {
 
-    //MARK: @IBOutlets
-    
+    // MARK: @IBOutlet(s)
+
     @IBOutlet var contentView: UIView!
     @IBOutlet weak var actionButton: UIButton!
     @IBOutlet weak var message: UILabel!
     @IBOutlet weak var title: UILabel!
-    
-    //MARK: @IBAction(s)
-    
+
+    // MARK: @IBAction(s)
+
     @IBAction func retryPressed(_ sender: Any) {
         onActionButtonTouched?()
     }
-    
-    // MARK: - Properties
-   
+
+    // MARK: Properties
+
     var errorMessage: String = "" {
         didSet {
             title.textColor = UIColor.DvtBlueColor
             message.textColor = UIColor.gray
             message.text = errorMessage
         }
     }
-    
-    
+
     var onActionButtonTouched: (() -> Void)? {
         didSet {
             actionButton.isHidden = onActionButtonTouched == nil
         }
     }
-    
-    // MARK: - Lifecycle
-    
+
+    // MARK: Lifecycle
+
     required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
         commonInit()
     }
-    
+
     override init(frame: CGRect) {
         super.init(frame: frame)
         commonInit()
     }
-    
+
     func commonInit() {
         let nib = UINib(nibName: String(describing: type(of: self)), bundle: nil)
-        contentView = nib.instantiate(withOwner: self, options: nil)[0] as! UIView
+        contentView = nib.instantiate(withOwner: self, options: nil)[0] as? UIView
```

Comment:
what if content view is nil from the casting to UIView, it shouldn't be added to subview if it is nil so maybe wrap it in an if let.

Reply:
done